packages/daos/mongo/src/dao/index.ts

```diff
@@ -1,81 +1,87 @@
-import path from 'path';
 import {
-  Db,
   InsertOneWriteOpResult,
-  MongoClient,
   UpdateWriteOpResult,
   FindOneOptions,
   DeleteWriteOpResultObject,
   IndexOptions,
 } from 'mongodb';
+import { Lambda } from 'aws-sdk';
+import { JsonServerless } from '@restlessness/core';
+
+interface ProxyRequest {
+  collectionName: string
+  operation: string
+  args: any[]
+}
 
 class MongoDao {
-  mongoClient: MongoClient
-  db: Db
+  mongoProxy: Lambda
+  proxyFunctionName = '_serverless-mongo-proxy'
 
   constructor() {
-    this.mongoClient = null;
-    this.db = null;
+    this.mongoProxy = new Lambda(
+      process.env['IS_OFFLINE'] ? { endpoint: 'http://localhost:4000' } : {});
   }
 
-  checkConnection() {
-    if (this.mongoClient === null) {
-      throw new Error('Mongo connection not initialized');
+  async invokeProxy(request: ProxyRequest) {
+    let invocationResult;
+    try {
+      await JsonServerless.read();
+      const serviceName = JsonServerless.service;
+      let stage = process.env['ENV_NAME'];
+      if (!stage) {
+        console.warn('dao-mongo stage name not set! defaulting to dev');
+        stage = 'dev';
+      }
```

Comment:
This is wrong. From ENV_NAME (that can be test, dev, staging and production) we read the EnvConfig from envs.json file, and from that we have something like:
`{
    "id": "staging",
    "type": "deploy",
    "stage": "dev"
  },
`
where we can read the "stage" property.